zkevm-circuits/src/witness/tx.rs

```diff
@@ -120,6 +120,9 @@ impl Transaction {
 
     /// Sign data
     pub fn sign_data(&self) -> Result<SignData, Error> {
+        if self.r.is_zero() && self.s.is_zero() && self.v == 0 {
```

Comment:
https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm#Signature_verification_algorithm
>  Verify that r and s are integers in [ 1 , n − 1 ] [1,n-1]. If not, the signature is invalid.

change to
`self.r.is_zero || self.s.is_zero()` ?